snuba/datasets/schema.py

```diff
@@ -118,3 +126,62 @@ def __init__(self, local_table_name, dist_table_name, columns,
 
     def _get_engine_type(self):
         return "ReplacingMergeTree(%s)" % self.__version_column
+
+
+class SummingMergeTreeSchema(MergeTreeSchema):
+
+    def _get_engine_type(self):
+        return "SummingMergeTree()"
+
+
+class MaterializedViewSchema(Schema):
+
+    def __init__(
+            self,
+            local_table_name,
+            dist_table_name,
```

Comment:
If this is the name of the materialized view, I would call it materialized_view_name or something similar. Otherwise it is very confusing for the reader to understand which table name is used for what.